README.md

```diff
@@ -1,6 +1,51 @@
-# CPU Power Manager for Gnome Shell
+<h1 align="center">CPU Power Manager for Gnome Shell</h1>
+<p align="center">
+  <a href="https://extensions.gnome.org/extension/945/cpu-power-manager/">
+    <img alt="Get it on GNOME Extensions" width="228" src="https://raw.githubusercontent.com/andyholmes/gnome-shell-extensions-badge/master/get-it-on-ego.svg?sanitize=true"/>
+  </a>
+  <br>
+  <a href="https://github.com/martin31821/cpupower/actions?query=workflow%3A%22latest%20build%22">
+    <img src="https://github.com/martin31821/cpupower/workflows/latest%20build/badge.svg" alt="latest build">
+  </a>
+  <a href="https://github.com/martin31821/cpupower/actions?query=workflow%3A%22release%20build%22">
+    <img src="https://github.com/martin31821/cpupower/workflows/release%20build/badge.svg" alt="release build">
+  </a>
+  <a href="https://github.com/martin31821/cpupower/releases">
+    <img alt="Lastest release" src="https://img.shields.io/github/v/release/martin31821/cpupower?label=latest%20release&sort=semver">
+  </a>
+  <img alt="Gnome" src="https://img.shields.io/badge/gnome-3.36-blue?logo=gnome&logoColor=white">
+  <a href="https://github.com/martin31821/cpupower/blob/master/LICENSE">
+    <img alt="License" src="https://img.shields.io/github/license/martin31821/cpupower.svg">
+  </a>
+  <a href="http://makeapullrequest.com">
+    <img alt="PRs Welcome" src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" target="_blank" />
+  </a>
+  <br>
+  <i>Gnome-Shell Extension for intel-pstate driver</i>
+  <br><br>
+  <a href="#prerequisites">Prerequisites</a> •
+  <a href="#installation">Installation</a> •
+  <a href="#translating">Translating</a> •
+  <a href="#packaging">Packaging</a> •
+  <a href="#developing">Developing</a>
+</p>
 
-Gnome-Shell Extension for intel-pstate driver.
+## Prerequisites
+
+You need to have an **Intel Core i CPU** of at least the **second Generation (2xxx Model Number)** and the following
+software installed to use this extension:
+
+- Gnome (since it's an extension)
+- Policykit (ships default with gnome)
+- intel_pstate needs to be enabled in the kernel (should be the default in almost all Distros)
+- bash
```

Comment:
For Ubuntu 20.04 installed with the "minimal" configuration, you need to install `gnome-shell-extension-prefs` by hand (at current state)

Reply:
Yes, that's unfortunately true. This is a manual dependency for all gnome-shell extensions under Ubuntu 20.04 which have a "Settings" entry.

Reply:
Fixed in 070f792